regexes/device/mobiles.yml

```diff
@@ -4499,7 +4502,7 @@ Vivo:
       model: 'iQOO U5'
     - regex: 'V2180GA'
       model: 'iQOO U5x'
-    - regex: '(?:V2049A|I2009)(?:[);/ ]|$)'
+    - regex: '(?:V2049A|I2009|iQOO 7 ?(?:India|Legend)?)(?:[);/ ]|$)'
```

Comment:
I don't think `iQOO 7 India` and `iQOO 7 Legend` are the same models with `iQOO 7`.

I see slightly differences for size, battery and connectivity.

`https://gadgetsnow.indiatimes.com/compare-mobile-phones/iqoo-7-vs-iqoo-7-legend`

Reply:
`iQOO 7 India` is a separate model. `iQOO 7` is same as `iQOO 7 Legend`.

https://www.gsmarena.com/vivo_iqoo_7-10674.php
https://www.gsmarena.com/vivo_iqoo_7_(india)-10859.php